packages/cli/test/helpers/prepare.js

```diff
@@ -536,15 +537,454 @@ module.exports = async function prepare(session, binaryPath, tmpFixturesDir) {
     },
   };
 
-  for (const [typeName, needed] of Object.entries(spec)) {
-    const directory = join(tmpFixturesDir, typeName);
+  for (const [fixtureName, fixtureContents] of Object.entries(spec)) {
+    prepare(tmpFixturesDir, fixtureName, fixtureContents);
+  }
+}
+
+const fixtures = {
```

Comment:
this list of fixtures is all new. follows same format as previous fixture list so that it can be used with the same `prepare` function logic

Reply:
I'm open to the idea of changing this data structure to something that allows more metadata maybe. Something along the lines of:

```js
const fixtures = [
  {
    name: 'monorepo-detection-turbo',
    content: [
      { name: '.vercel/project.json', data: JSON.stringify({ /* ... */ }), },
      /* ... */
    ],
  },
]
```